appconfig/settings/filealerter.q

```diff
@@ -0,0 +1,8 @@
+\d .fa
+
+decodepcaps:0b                   // Whether to connect to tickerplant when the file alerter process starts
```

Comment:
there seems to be a disconnect between the variable name & the comment here - I assume connecting to tickerplant is more of a "side effect" of enabling pcap decoding? probably worth clarifying this comment a little, potential for confusion here I think

Reply:
`decodepcaps` is there so that the filealerter process doesn't connect to the tickerplant on startup if nothing is to be done with pcap files. So when `processpcaps` is run it can't send the packets table to the tickerplant and filelalerter doesn't needlessly connect to the tickerplant. 
It's also just been added to `processpcaps` to avoid errors when trying to send to the tickerplant if there's no connection
```
processpcaps:{[path;file]
        if[not decodepcaps; :()];
        .lg.o[`alerter;"processing pcap file"];
```
Would something like this be better?
```
When true, filealerter connects to tickerplant on startup and processpcaps sends parsed pcaps to tickerplant. processpcaps returns an empty list when false
```
or should the part about connecting to the tickerplant be removed? May be confusing to leave it out as the filealerter process would have to be restarted if `decodepcaps` was false and someone wanted to start sending the packets table to the tickerplant while filealerter is already running

Reply:
I think something like

```q
// parse PCAPs found by filealerter (causes connection to TP)
```

is descriptive and succinct